test/WebSites/ViewComponentWebSite/Startup.cs

```diff
@@ -15,7 +16,13 @@ public void Configure(IApplicationBuilder app)
             {
                 services.AddMvc(configuration);
             });
-            app.UseMvc();
+
+            app.UseMvc(routes =>
+            {
+                routes.MapRoute(
+                    name: "default",
+                    template: "{controller=Home}/{action=Index}/{id?}");
```

Comment:
> we tend to write demo-ware rather than realistic test sites

This one is definitely true. Our test sites aren't nearly close to the level of complexity that a real site would be, and are often focused on other details than making pretty URLs.